### ISSUE TITLE
cmd/subd: cpu is spinning when the rootfs is small

### ISSUE THREAD

**Issue by masiulaniec** [Symantec/Dominator#419]
`subd` is consuming 70% of a single core of my test VM. I have started it as `subd -rootDir=/test` where /test is an empty directory. Adding `-showStats` reveals that many scan cycles are being started in short succession.

On inspection I think `sub/scanner.scannerDaemon` needs something like a `time.Ticker` to place an upper bound on the frequency of scans.

I'm running 42542a1cbff26005b55947ae5db6829422536b45.

**Comment by rgooch** [comment]
I've added https://github.com/rgooch/Dominator/commit/953c129c09d3c4227e18b175a577325e643396f6 which enforces a minimum one second scan cycle. It worked for me. Please try it out. If this works for you, I'll merge upstream.

**Comment by masiulaniec** [comment]
Thanks for the quick response. I have already applied the following on my side:

```
@@ -38,7 +39,8 @@ func scannerDaemon(rootDirectoryName string, cacheDirectoryName string,
        runtime.LockOSThread()
        loweredPriority := false
        var oldFS FileSystem
-       for {
+       tick := time.Tick(5*time.Second)
+       for ;; <-tick {
                fs, err := scanFileSystem(rootDirectoryName, cacheDirectoryName,
                        configuration, &oldFS)
                if err != nil {
```

It fixes the problem. I've tried your diff and it works equally well.

**Comment by rgooch** [comment]
Fixed by PR #421
